[TW-1534] Refresh /docs/collections/collections-overview (#5352)
postmanlabs/postman-docs: src/pages/docs/collections/collections-overview.md

---

> title: "Postman Collections overview"

Reviewer: Shouldn't we leave this title as `Create, use, and run Postman Collections`? We discussed with Ashley on her PR earlier today about having long, descriptive titles for overviews. That's what we do in other locations

> title: "Postman Collections overview"

Author: ```suggestion
title: "Organize and automate API requests in Postman Collections"
```

> title: "Postman Collections overview"

Author: done

    url: "https://www.postman.com/case-studies/paypal/"
---

> A _Postman collection_ is a set of API endpoints or requests, along with each endpoint's authorization type, parameters, headers, request bodies, tests, and settings grouped under the same collection ID.

Reviewer 2: I think Collection is capitalized when preceded by Postman. More details at https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/4604690433/Branded+Terms

Author: yes, you are right!

> A collection allows you to group requests with different method types (for example `GET`, `POST`, `DELETE`, and `PUT`) and organize them into folders or subfolders. You can share collections with team members, as well as import and export them into other Postman instances.

Reviewer: ```suggestion
A collection enables you to group requests with different method types (for example `GET`, `POST`, `DELETE`, and `PUT`) and organize them into folders or subfolders. You can share collections with team members, as well as import and export them into other Postman instances.
```

> The _Collection Format_ is the specification that drives collections in Postman. It's:

Reviewer: ```suggestion
Postman Collections are based on the open source _collection format_ that makes it possible to share and run collections. The collection format is:
```

Reviewer: Collection format isn't a branded term, so I would leave as lower case (though you would need to capitalize collection if it's preceded by Postman).

Also a suggestion: How about adding this sentence and the following two bullets and the sentence "To learn more about the format..." to a separate section called "Postman Collection format" following "Get started using collections"

* Portable and provides a unique interface for organizing API requests and modeling API workflows.
* Machine and human readable and can be used to generate client and server-side SDKs, documentation, and mock servers.

## Get started using collections

> To learn more about the format, see the [Postman Collection Format documentation](https://learning.postman.com/collection-format/).

Reviewer 2: ```suggestion
To learn more about the collection format, see the [Postman Collection Format documentation](https://learning.postman.com/collection-format/).
```

If you're getting started with collections for the first time, learn how to [Create your first collection](/docs/getting-started/first-steps/creating-the-first-collection/).

After you've created collections, you can learn more about how to manage and use them in [Create and manage request collections in Postman](/docs/collections/using-collections/).

> You'll also learn how the _Collection Runner_ enables you to run a collection's requests in a specified sequence. You can run collections manually, on a schedule, from a monitor, from a webhook, or in your CI/CD pipeline by running it from the command line. See [Using the Collection Runner](/docs/collections/running-collections/running-collections-overview/) for more information.

Reviewer: ```suggestion
You'll also learn how the _Collection Runner_ enables you to run a collection's requests in a specified sequence. You can run collections manually, on a schedule, from a monitor, from a webhook, or in your CI/CD pipeline by running it from the command line. See [Test your API functionality](/docs/collections/running-collections/running-collections-overview/) for more information.
```

    url: "https://www.postman.com/case-studies/paypal/"
---

> A _Postman collection_ is a set of API endpoints or requests, along with each endpoint's authorization type, parameters, headers, request bodies, tests, and settings grouped under the same collection ID.

Reviewer: ```suggestion
A _Postman Collection_ is a set of API endpoints or requests, along with each endpoint's authorization type, parameters, headers, request bodies, tests, and settings grouped under the same collection ID.
```

Reviewer: Note that Postman Collection is capitalized (while collection on its own isn't).

Also, I think it would be good to add another sentence to this first paragraph that tells readers the benefits of collections or why they'd want to use them. For example, the previous text described that collections make it easy to find and reuse your favorite requests. This would help bet the featured snippet (first paragraph) to the preferred 40-50 word range.
